4851 - Fix dropdown focus and arrow keys with datagrid (#5172)
infor-design/enterprise: docs/CHANGELOG.md

- `[Datagrid]` Fixed an issue where stretching the last column of a table was not consistent when resizing the window. ([#5045](https://github.com/infor-design/enterprise/issues/5045))
- `[Datagrid]` Fixed an issue where time format HHmm was not working for time picker editor. ([#4926](https://github.com/infor-design/enterprise/issues/4926))
- `[Datagrid]` Fixed an issue where setting stretchColumn to 'last' did not stretch the last column in the table. ([#4913](https://github.com/infor-design/enterprise/issues/4913))

> - `[Datagrid]` Fixed an issue where dropdown focus and arrow key was moving across the grid columns, which let open multiple dropdowns. ([#4851](https://github.com/infor-design/enterprise/issues/4851))

Reviewer: ```suggestion
- `[Datagrid]` Fixed an issue where when focusing dropdowns and then using arrow key, it would move across the grid columns leaving multiple open dropdowns. ([#4851](https://github.com/infor-design/enterprise/issues/4851))
```
